docs/tutorials/dev.md

```diff
@@ -14,7 +14,7 @@ cd elasticdl
 We prefer to install all building tools in a Docker image.
 
 ```bash
-docker build --target dev -t elasticdl:dev -f elasticdl/docker/Dockerfile .
+sh scripts/travis/build_images.sh
```

Comment:
How about we chmod +x build_images.sh, so we don't need to type `sh` here?

Reply:
Good suggestion!